lib/package.gi

```diff
@@ -1348,9 +1348,14 @@ InstallGlobalFunction( ReadPackage, function( arg )
       Read( filename );
       LEAVE_NAMESPACE();
       return true;
-    else
-      return false;
+    elif error then
+      Print( "ReadPackage could not read <", pkgname, ">/", relpath, "\n" );
```

Comment:
```suggestion
      # TODO: turn this into an `Error` in a future GAP version
      Print( "ReadPackage could not read <", pkgname, ">/", relpath, "\n" );
```

Reply:
Also, should we use `Info(InfoWarning, 1, "...")` instead? Or `Print` but with some prefix like `#I  ` ?